src/main/java/frc/robot/Robot.java

```diff
@@ -310,30 +277,14 @@ public void teleopSetup() {
 
   }
   /** This function is called periodically during operator control. */
+  TalonSRX turretMotor = new TalonSRX(RobotMap.TURRET_PORT);
```

Comment:
This should be declared at the top of the file and I think it's already initialized somewhere, so this should not be re-initializing it.